tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/wifi.py

```diff
@@ -32,6 +32,7 @@ class Wifi(Base):
     _manage_certs_screen_locator = (By.ID, 'wifi-manageCertificates')
     _import_certs_locator = (By.CSS_SELECTOR, '[data-l10n-id="importCertificate"]')
     _select_certs_screen_locator = (By.ID, 'wifi-selectCertificateFile')
+    _select_active_wifi = (By.CSS_SELECTOR, '.wifi-availableNetworks li.active')
```

Comment:
All our locators have '_locator' at the end, so you should do that too.
(I don't like this convention myself that much, but that's how it's done, so we should stick with it)